### ISSUE TITLE
chapter 7 cbow loss outputs 'nan'

### ISSUE THREAD

**Issue by Xnsam** [nfmcclure/tensorflow_cookbook#108]
Hi ,

I am getting loss result as nan while implementing the cbow code from chapter 7.
I wrote the code learning it from the book and then suspicious of my output i copy pasted from repo...
and still i got 'nan'
Please kindly suggest a work around...

**Comment by Xnsam** [comment]
I changed learning_rate = 0.5 to learning rate = 0.01 also tried with learning_rate = 0.05 and batch_size=100, after 50k epochs it gave me loss=3.4292  from learning_rate= 0.01 and loss=2.6028  from learning_rate=0.05...with a gradual decrease from both the cases...need to optimize it though...
